components/avatar/avatar.html

```diff
@@ -0,0 +1,149 @@
+    
+    <!-- External Imports -->
+    <link rel="stylesheet" href="avatar.css">
+    <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body>
+    <section class="section-center">
+        <!-- navbar -->
+        <nav class="navbar flex items-center justify-between pt-1 section-center z-5">
+            <a class="humburger-icon icon-open text-lg"><i class="fas fa-bars ml-5"></i></a>
+            <a class="humburger-icon icon-close text-lg displayNone"><i class="fas fa-bars ml-5"></i></a>
+            <a href="/index.html" class="mr-5"><img src="/assests/logo-primary.png" alt="NOMARK" class="nomark-logo"></a>
+        </nav>
+        <!-- side-bar -->
+        <div class="sidebar sidebarRemove flex flex-col scroll-style z-4 pr-7 p-2">
+            <h3 class="sidebar-heading text-lg">Getting Started</h3>
+            <ul>
+                <li><a href="/documentation/doc.html" class="sidebar-items">Installation</a></li>
+            </ul>
+            <h3 class="sidebar-heading text-lg">Components</h3>
+            <ul>
+                <li><a href="avatar.html" class="sidebar-items active">Avatar</a></li>
+                <li><a href="" class="sidebar-items">Alert</a></li>
+                <li><a href="/components/badge/badge.html" class="sidebar-items">Badge</a></li>
+                <li><a href="" class="sidebar-items">Button</a></li>
+                <li><a href="" class="sidebar-items">Card</a></li>
+                <li><a href="" class="sidebar-items">Modal</a></li>
+            </ul>
+        </div>
+        <!-- Square Avatar -->
+        <main class="click-remove my-7">
+            <div class="m-2">
+                <h2 class="text-xl font-bolder">Avatar</h2>
+                <p class="text-ml py-5">Avatar is used to represent the users profile picture.</p>
+            </div>
+            <div class="component-container flex flex-col">
+                <div class="component-heading p-2">
+                <h3 class="font-bold text-lg">Square Avatar</h3>
+                <p>Square Avatar has 5 different sizes you can use avatar-sm for small image, avatar-md for medium, avatar-lg for large and so onn. </p>
+            </div>
+                <div class="component-display flex justify-center p-5">
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-sm m-4">
+                    </figure>
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-md m-4">
+                    </figure>
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-lg m-4">
+                    </figure>
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-xl m-4">
+                    </figure>
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-xxl m-4">
```

Comment:
Provide alternate text with respect to your image.

Reply:
Done with the alt text